v2/policyfile.go

```diff
@@ -0,0 +1,213 @@
+package tailscale
```

Comment:
Named this file policyfile to match the api docs section naming?

Reply:
Correct.